ament_uncrustify/ament_uncrustify/main.py

```diff
@@ -72,6 +77,20 @@ def main(argv=sys.argv[1:]):
         print("Could not config file '%s'" % args.config_file, file=sys.stderr)
         return 1
 
+    if args.linelength is not None:
+        # check if different from config file
+        config = ConfigParser()
+        with open(args.config_file, 'r') as h:
+            config_str = h.read()
+        config.read_string('[DEFAULT]\n' + config_str)
+        code_width = config['DEFAULT']['code_width']
+        code_width = int(re.split('[ \t#]', code_width, maxsplit=1)[0])
+        if args.linelength != code_width:
+            # generate temporary config file with custom line length
+            temp_config = tempfile.NamedTemporaryFile('w')
```

Comment:
https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile

> Whether the name can be used to open the file a second time, while the named temporary file is still open, varies across platforms (it can be so used on Unix; it cannot on Windows NT or later).

Fixed in a5c857c4e1ca8f89a835dc7559d3c371b14fa596.